website/src/utils/yamlDataUtils/rename.js

```diff
@@ -79,31 +98,35 @@ const renameDBConnEvents = (path, oldValue, newValue, yamlData) => {
     delete envTarget[`${oldValue.key.toUpperCase()}_PASSWORD`]
     delete envTarget[`${oldValue.key.toUpperCase()}_USER`]
     delete envTarget[`${oldValue.key.toUpperCase()}_DB`]
-    // DB Connection
+    // Connection
     const connSteps = ['core_lib', 'connections', path.at(-2)]
     const connTarget = getValueAtPath(data, connSteps)
-    if(connTarget.url.protocol !== 'sqlite'){
-        connTarget['url']['host'] = '${oc.env:' + newHost + '}'
-        connTarget['url']['port'] = '${oc.env:' + newPort + '}'
-        connTarget['url']['password'] = '${oc.env:' + newPassword + '}'
-        connTarget['url']['username'] = '${oc.env:' + newUser + '}'
-        connTarget['url']['file'] = '${oc.env:' + newDB + '}'
+    if(connTarget.config.url.protocol !== 'sqlite'){
```

Comment:
class as type

Reply:
here we need to specifically check for `sqlite` only